setup.py

```diff
@@ -211,27 +217,44 @@ def write_version(filename=os.path.join(*['airflow',
 qds = ['qds-sdk>=1.10.4']
 rabbitmq = ['librabbitmq>=1.6.1']
 redis = ['redis>=2.10.5,<3.0.0']
-s3 = ['boto3>=1.7.0, <1.8.0']
 salesforce = ['simple-salesforce>=0.72']
 samba = ['pysmbclient>=0.1.3']
 segment = ['analytics-python>=1.2.9']
 sendgrid = ['sendgrid>=5.2.0']
 slack = ['slackclient>=1.0.0']
 mongo = ['pymongo>=3.6.0']
-snowflake = ['snowflake-connector-python>=1.5.2',
-             'snowflake-sqlalchemy>=1.1.0']
-ssh = ['paramiko>=2.1.1', 'pysftp>=0.2.9', 'sshtunnel>=0.1.4,<0.2']
+snowflake = [
+    'snowflake-connector-python>=1.5.2',
+    'snowflake-sqlalchemy>=1.1.0',
+]
+ssh = [
+    'paramiko>=2.1.1',
+    'pysftp>=0.2.9',
+    'sshtunnel>=0.1.4,<0.2',
+]
 statsd = ['statsd>=3.0.1, <4.0']
 vertica = ['vertica-python>=0.5.1']
 webhdfs = ['hdfs[dataframe,avro,kerberos]>=2.0.4']
 winrm = ['pywinrm==0.2.2']
 zendesk = ['zdesk']
 
-all_dbs = postgres + mysql + hive + mssql + hdfs + vertica + cloudant + druid + pinot \
-    + cassandra + mongo
+all_dbs = (
+    postgres +
+    mysql +
+    hive +
+    mssql +
+    hdfs +
+    vertica +
+    cloudant +
+    druid +
+    pinot +
+    cassandra +
```

Comment:
Generally I try to avoid unrelated white-space only/formatting changes as means that `git blame` will show this commit as the last change - just makes it harder to find the original source of a line)

Reply:
I want to enter the alphabetical order in this file when they have a large number of elements that are printed one by one in a line. I want to introduce it just for the git blame. In the future it will be easier to find where this change comes from. Unfortunately, there must be a change sometime in order for it to be better tomorrow.

Reply:
I will introduce the proposed changes today.

Reply:
I'm done. I hope I did not break something.